style/ruby/README.md

```diff
@@ -0,0 +1,14 @@
+# Ruby Style Enforcement
+
+It is recommended that you use a ruby style enforcement tool such as [RuboCop](/bbatsov/rubocop). This can be integrated in your CI build and optionally use a service like HoundCI to comment on Pull Requests indicating violations.
+
+There is a RuboCop configuration file available in this directory that can be used as a base for a new project.  There are various modifications in that configuration that we have collaboratively determined are acceptable in DLSS.
```

Comment:
Should we have a paragraph about why style guides are useful? I worry slightly that it sounds like another metric we're trying to hit, but it's not just about making e.g rubocop happy, but using rubocop's guidance to write good, clear, testable code.